pyproject.toml

```diff
@@ -1,5 +1,10 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
 [project]
-name = "noisepy-seis-io"
+name = "noisepy_seis_io"
```

Comment:
I am not sure why it appeared this way, because I have initialized the name as noisepy-seis-io only, will create a new PR with the changes mentioned and check again if it appears this way.